app/workers/update_feature_metrics_dashboard.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateFeatureMetricsDashboard
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 0, queue: :low_priority
+
+  def perform
+    dashboard = FeatureMetricsDashboard.new
+    dashboard.load_updated_metrics
+    dashboard.save!
+  end
```

Comment:
Should we delete the old record as we go? One interesting (but not immediately useful) side effect of keeping these records is the historical view of these metrics over time.

Reply:
I would say keep the records, they are not costing us much.